src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -48,7 +48,49 @@ public class SendReceiveTest : ClientWebSocketTestBase
                     recvResult = await cws.ReceiveAsync(receiveSegment, ctsDefault.Token);
                 }
 
-                await cws.CloseAsync(WebSocketCloseStatus.NormalClosure, "PartialMessageTest", ctsDefault.Token);
+                await cws.CloseAsync(WebSocketCloseStatus.NormalClosure, "PartialMessageDueToSmallReceiveBufferTest", ctsDefault.Token);
+            }
+        }
+
+        [ActiveIssue(21102)]
```

Comment:
I've updated the corefx-net.cloudapp.net cloud service to handle the new endpoint using the `replyWithPartialMessages` query string parameter.

So, you can update this ActiveIssue to be:

```
[ActiveIssue(21102, TargetFrameworkMonikers.Uap)]
```